docs/plugins.md

```diff
@@ -13,9 +13,7 @@ Let's take a quick look at the current dokku nginx plugin that's shipped with do
 ## Installing a plugin
 
 ```shell
-cd /var/lib/dokku/plugins/available
-git clone <git url>
-dokku plugin:install
+dokku plugin:install <git url>
```

Comment:
I mean why would you need root for a directory that you'd expect to be owned by root?